impl/src/test/java/org/ehcache/docs/GettingStarted.java

```diff
@@ -362,6 +362,29 @@ public void cacheSerializingCopiers() throws Exception {
   }
 
   @Test
+  public void cacheCopierAtCacheManagerLevel() {
+    DefaultCopyProviderConfiguration copierType = new DefaultCopyProviderConfiguration()
+        .addCopierFor(Person.class, (Class)SerializingCopier.class);
```

Comment:
The test from the actual issue #605 is flawed, `Description` isn't `Serializable`... this will now throw early (rather than at "cache runtime" (i.e. anything that involves copying for the first time)